### PR TITLE
➕ Add cupertino activity indicator

### PR DESCRIPTION
**Pull request by YazeedAlKhalaf**
Related to flutter/flutter#72545

This PR adds Cupertino activity indicator in the testing app.

Note: The new name for activity indicator in progress view in SwiftUI.

Verified it is the the same size and everything is good. One difference may be the animation speed or when it starts. Other than that everything is alright.

| New addition |
| -- |
| <video src="https://user-images.githubusercontent.com/41989385/127160801-b677d25a-901a-4f75-9716-c666e3e8b491.mov" /> |

### CODE REVIEW

**Comment by YazeedAlKhalaf** [comment]
Can you review this @DanielEdrisian ?

File: ios_widget_catalog_compare/ios/.gitignore [hunk 1]
```diff
@@ -30,3 +30,6 @@ Runner/GeneratedPluginRegistrant.*
 !default.mode2v3
 !default.pbxuser
 !default.perspectivev3
+
+# misc
+build/
```
Comment (jmagman) [inline ios_widget_catalog_compare/ios/.gitignore:35]: Restore the end newline.
```suggestion
build/

```

**Review by jmagman — CHANGES_REQUESTED** [review]
> And to view the changes without the formatting and pubspec changes, view commit `73e21d2`. 😉

Can you revert the whitespace formatting changes?  Flutter style guide is 2 spaces for indentation.  And even if we wanted it changed, it should be done in its own commit.

**Comment by YazeedAlKhalaf** [comment]
@jmagman hopefully now everything is good.

Did the changes requested.
First time making a revert, that's why I had some changes I didn't intend to revert in the revert commit 😁

**Review by jmagman — APPROVED** [review]
Checked this out and tested it, LGTM, and thank you!

File: ios_widget_catalog_compare/ios/Runner/OverlaySwiftUIView.swift [hunk 2]
```diff
@@ -93,6 +93,12 @@ struct OverlaySwiftUIView: View {
           }
         })
       ),
+      "CupertinoActivityIndicator":
+        ("Cupertino Activity Indicator (Progress View)",
```
Comment (jmagman) [inline ios_widget_catalog_compare/ios/Runner/OverlaySwiftUIView.swift:97]:
```suggestion
        ("Cupertino Activity Indicator",
```

Comment (jmagman) [reply ios_widget_catalog_compare/ios/Runner/OverlaySwiftUIView.swift:97]: Sorry, what you have is fine, please ignore.

**Pull request merged**